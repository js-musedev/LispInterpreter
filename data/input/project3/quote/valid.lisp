(QUOTE (CONS 34 92))
(CDR (QUOTE (4 (CONS 45 6))))