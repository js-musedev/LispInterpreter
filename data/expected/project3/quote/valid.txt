(CONS 34 92)
